website/blog/2023-02-01-databend-weekly-79.md

```diff
@@ -0,0 +1,162 @@
+- support array concat ([#9804](https://github.com/datafuselabs/databend/pull/9804))
+
+**Query**
+
+- add topn runtime filter in native storage format ([#9738](https://github.com/datafuselabs/databend/pull/9738))
+- enable hashtable state pass from partial to final ([#9809](https://github.com/datafuselabs/databend/pull/9809))
+
+**Storage**
+
+- add pruning stats to EXPLAIN ([#9724](https://github.com/datafuselabs/databend/pull/9724))
+- cache bloom index object ([#9712](https://github.com/datafuselabs/databend/pull/9712))
+
+### Code Refactoring :tada:
+
+- 'select from stage' use ParquetTable ([#9801](https://github.com/datafuselabs/databend/pull/9801))
+
+**Meta**
+
+- expose a single "kvapi" as public interface ([#9791](https://github.com/datafuselabs/databend/pull/9791))
+- do not remove the last node from a cluster ([#9781](https://github.com/datafuselabs/databend/pull/9781))
+
+**AST/Expression/Planner**
+
+- unify Span and Result ([#9713](https://github.com/datafuselabs/databend/pull/9713))
+
+**Executor**
+
+- merge simple pipe and resize pipe ([#9782](https://github.com/datafuselabs/databend/pull/9782))
+
+### Bug Fixes :wrench:
+
+**Base**
+
+- fix not linux and macos jemalloc fallback to std ([#9786](https://github.com/datafuselabs/databend/pull/9786))
+
+**Config**
+
+- fix table_meta_cache can't be disabled ([#9767](https://github.com/datafuselabs/databend/pull/9767))
+
+**Meta**
+
+- when import data to meta-service dir, the specified "id" has to be one of the "initial_cluster" ([#9755](https://github.com/datafuselabs/databend/pull/9755))
+
+**Query**
+
+- fix and refactor aggregator ([#9748](https://github.com/datafuselabs/databend/pull/9748))
+- fix memory leak for data port ([#9762](https://github.com/datafuselabs/databend/pull/9762))
+- fix panic when cast jsonb to string ([#9813](https://github.com/datafuselabs/databend/pull/9813))
+
+**Storage**
+
+- fix up max_file_size may oom ([#9740](https://github.com/datafuselabs/databend/pull/9740))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### DML Command - UPDATE
+
+Modifies rows in a table with new values.
+
+> **Note:** Databend guarantees data integrity. In Databend, Insert, Update, and Delete operations are guaranteed to be atomic, which means that all data in the operation must succeed or all must fail.
+
+**Syntax**
+
+```sql
+UPDATE <table_name>
+SET <col_name> = <value> [ , <col_name> = <value> , ... ]
+    [ FROM <table_name> ]
+    [ WHERE <condition> ]
+```
+
+**Learn More**
+
+- [Docs | DML Command - UPDATE](https://databend.rs/doc/sql-commands/dml/dml-update)
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support Arrow Flight SQL Protocol
+
+Currently Databend support MySQL protocol, as an alternative to this Databend also should support Arrow Flight SQL protocol.
```

Comment:
Currently Databend supports the MySQL protocol, and it would be great if Databend could support the Arrow Flight SQL protocol as well.